### PR TITLE
MRG: Add HRTF Pair Files

### PR DESCRIPTION
**Pull request by maddycapp27**
Add files containing HRTF magnitudes and unwrapped phase for use when interpolating between HRTFs. This needed to be created to manually correct unpredictable unwrapping errors that lead to incorrect interpolation between some angle pairs.

Each file contains unwrapped phase, amplitude, sampling frequency, the list of angles from the original hrtfs, and a list of each of the arrays of pairs (as phase and amplitude are stored as pairs).

### CODE REVIEW

**Comment by rkmaddox** [comment]
Almost there--just noticed a couple things still to be fixed. We will ping when ready.

**Comment by maddycapp27** [comment]
Hi @drammock @Eric89GXL 
I think all the wrinkles with these two files have been worked out on my end. There's one for each sampling frequency and both use the cipic library (I don't think it makes sense to let people interpolate with the barb hrtfs). Thanks much!

**Comment by larsoner** [comment]
> I don't think it makes sense to let people interpolate with the barb hrtfs

That's fine, just make sure it is clearly documented that interp only works for "cipic" and it throws an error (add a test for this) if they try to interpolate with "barb".

**Comment by larsoner** [comment]
Can you add a README.rst or README.md in the `hrtf` directory that specifies what is in these files / how they were processed? A few sentences is enough.

**Comment by maddycapp27** [comment]
@Eric89GXL Is that sufficient as documentation? Or is there a need/way to include the code from which the file was generated?

**Comment by larsoner** [comment]
If there is a code, you can put it up on `gist` and link to it or put it here, up to you. I assumed it was a bunch of manual adjustments.

**Comment by rkmaddox** [comment]
@maddycapp27, are the new, fixed data uploaded? If so, we should ask to have this PR reviewed, since the one over in ``expyfun`` won't pass its tests until this one is merged.

**Comment by maddycapp27** [comment]
@rkmaddox Couldn't remember so I just made sure they were as up to date as possible.

@Eric89GXL @drammock Would you be willing to review this at your convenience? Much appreciated!

File: hrtf/README.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+These files each contain all fo the information necessary to interpolate
```
Comment (larsoner) [inline hrtf/README.rst:1]: fo

**Comment by larsoner** [comment]
Good enough!

**Comment by rkmaddox** [comment]
Thanks @Eric89GXL

**Pull request merged**